ihmc-footstep-planning/src/test/java/us/ihmc/footstepPlanning/log/FootstepPlannerLoggerTest.java

```diff
@@ -1,53 +1,51 @@
 package us.ihmc.footstepPlanning.log;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import toolbox_msgs.msg.dds.AStarBodyPathPlannerParametersPacket;
 import toolbox_msgs.msg.dds.FootstepPlannerParametersPacket;
 import toolbox_msgs.msg.dds.FootstepPlanningRequestPacket;
 import toolbox_msgs.msg.dds.FootstepPlanningToolboxOutputStatus;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-import toolbox_msgs.msg.dds.VisibilityGraphsParametersPacket;
+import toolbox_msgs.msg.dds.SwingPlannerParametersPacket;
 import us.ihmc.commons.time.Stopwatch;
 import us.ihmc.euclid.geometry.Pose3D;
 import us.ihmc.euclid.tuple4D.Quaternion;
 import us.ihmc.footstepPlanning.FootstepPlannerOutput;
 import us.ihmc.footstepPlanning.FootstepPlannerRequest;
 import us.ihmc.footstepPlanning.FootstepPlanningModule;
+import us.ihmc.footstepPlanning.log.FootstepPlannerLogLoader.LoadResult;
 import us.ihmc.footstepPlanning.tools.FootstepPlannerMessageTools;
 import us.ihmc.footstepPlanning.tools.PlanarRegionToHeightMapConverter;
 import us.ihmc.pathPlanning.DataSet;
 import us.ihmc.pathPlanning.DataSetIOTools;
 import us.ihmc.pathPlanning.DataSetName;
 import us.ihmc.robotics.robotSide.RobotSide;
 import us.ihmc.sensorProcessing.heightMap.HeightMapMessageTools;
+import us.ihmc.tools.IHMCCommonPaths;
 
 import java.io.File;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 public class FootstepPlannerLoggerTest
 {
-   private static final String logDirectory = System.getProperty("user.home") + File.separator + "testLog" + File.separator;
-
-   @AfterAll
-   public static void cleanTestDirectory()
-   {
-      FootstepPlannerLogger.deleteOldLogs(0, logDirectory);
-   }
-
    /**
-    * This test ensures that logging the footstep plan's doesn't take too long. The speed in which this takes doesn't need ot be exact but
-    * this allows us to ensure that logging of footstep plan's isn't slow. The will slow down threads if planning over and over again
+    * We use an epsilon here because when data gets logs, it may get slightly rounded depending on the situation.
+    * So we want to check that things haven't changed
+    * a ton
     */
-   @Disabled
-   @Test
-   public void testFootstepLoggingSpeed()
+   private static final double EPSILON = 1e-5;
+   private static final String logDirectory = System.getProperty("user.home") + File.separator + "testLog" + File.separator;
+   private final FootstepPlannerRequest request = new FootstepPlannerRequest();
+   private final FootstepPlanningModule planningModule = new FootstepPlanningModule("TestModule");
```

Comment:
I'm not sure if it's still the case but I know unit tests used to run multi-threaded on the bamboo, so having fields like this shared between tests would possibly cause problems.

Reply:
Ya I haven't run into this issue on Github, seems to work currently